packages/babel-preset/index.js

```diff
@@ -27,13 +27,7 @@ const PRESET_ENV_OPTIONS = [
   'shippedProposals',
 ];
 
-const DEFAULT_BROWSERS = [
-  'ie >= 11',
```

Comment:
we definitely don't want any IE support by default?

don't all of our apps set custom browser lists anyway?

Reply:
Apps usually (should) set a default via a browserslist config file, yeah. I think that generally all of our libraries should default to a target that balances minimal compiled output and browser support. Including IE, though makes most of `bugfixes` useless since there is very little that can be fixed without  down-leveling. 

I think my position here is that libraries should not exclude ie11 support, e.g. should allow further compiling and polyfilling in the app, like we do for the browser support pages. I don't think a default set of browsers that includes ie11 here tho makes that more likely tho. Overall, new syntax can generally be polyfilled/down-leveled, the real issues are usually around subtle CSS differences or engine quirks, things that you don't uncover unless you are actually testing in IE